sinusbot_installer.sh

```diff
@@ -62,7 +62,7 @@ function checkInstall {
 # Update notify
 
 cyanMessage "Checking for the latest latest installer version"
-LATEST_VERSION=$(wget -q --timeout=60 -O - https://raw.githubusercontent.com/SinusBot/installer-linux/master/sinusbot_installer.sh | grep -Po '(?<=Instversion=")([0-9]\.[0-9]\.[0-9]+)')
+LATEST_VERSION=$(wget -q --no-check-certificate --timeout=60 -O - https://raw.githubusercontent.com/SinusBot/installer-linux/master/sinusbot_installer.sh | grep -Po '(?<=Instversion=")([0-9]\.[0-9]\.[0-9]+)')
```

Comment:
Maybe the better way is to update the ca-certificates at the beginning of the script instead of adding the `--no-check-certificate` every time.